projects/igniteui-angular/src/lib/date-range-picker/date-range-picker.component.ts

```diff
@@ -915,13 +917,23 @@ export class IgxDateRangePickerComponent extends DisplayDensityBase
     }
 
     private initialSetValue() {
-        // if there is no value, no ngControl but we have inputs we may have value set trough
-        // inputs' ngModels - we should generate our initial control value
-        if (!this.value && this.hasProjectedInputs && !this._ngControl) {
-            const start = this.projectedInputs.find(i => i instanceof IgxDateRangeStartComponent).dateTimeEditor.value;
-            const end = this.projectedInputs.find(i => i instanceof IgxDateRangeEndComponent).dateTimeEditor.value;
-            this.updateValue({ start, end });
-
+        // if there is no value and no ngControl on the picker but we have inputs we may have value set through
+        // their ngModels - we should generate our initial control value
+        if ((!this.value || (!this.value.start && !this.value.end)) && this.hasProjectedInputs && !this._ngControl) {
+            const start = this.projectedInputs.find(i => i instanceof IgxDateRangeStartComponent);
+            const end = this.projectedInputs.find(i => i instanceof IgxDateRangeEndComponent);
+            const value = {
+                start: start.dateTimeEditor.value,
+                end: end.dateTimeEditor.value
+            };
+
+            if (start.dateTimeEditor.ngControl && end.dateTimeEditor.ngControl) {
```

Comment:
this check is not necessary. Go ahead and set the field like this:
```
this._value = value;
```
Calling `updateValue` will do the same and will set the values of the projected inputs. But you just took the values from them. Actually you can simplify this to:
```
this._value = {
    start: start.dateTimeEditor.value,
    end: end.dateTimeEditor.value
};
```